src/sql/workbench/contrib/notebook/browser/cellToolbarActions.ts

```diff
@@ -132,10 +132,6 @@ export class CellToggleMoreActions {
 	private removeDuplicatedAndStartingSeparators(actions: (Action | CellActionBase)[]): void {
```

Comment:
This kind of thing is perfect for unit testing *hint hint* 😁

Reply:
hehe. I can add a thing :)